website/src/pages/components/form/index.js

```diff
@@ -155,20 +160,25 @@ function FormPage() {
       return (
         <Container padding="8">
           <Form initialValues={initialValues} onSubmit={onSubmit}>
-            <Stack gap="8">
-              <Text as="h2" textStyle="heading4">About you</Text>
-              <Input name="name" label="Name" />
-              <Select name="relationshipStatus" label="Relationship status" options={relationshipStatusOptions} />
-              <Dropdown name="favouriteMovie" label="Favourite movie" placeholderComponent={MoviePlaceholder} options={movieOptions} optionToString={movieOptionToString} optionComponent={MovieOption}  />
-              <CheckboxGroup name="fruits" label="Which fruits do you like?" options={fruitOptions} />
-              <Checkbox label="Do you like ice cream?" hideLabel name="likeIceCream" helpText="You MUST like it!">I like ice cream</Checkbox>
-              <RadioGroup name="hungry" label="Are you hungry?" options={hungryOptions} />
-              <Frequency name="salary" label="Salary (before tax)" amountPrefix="$" />
-              <DatePicker name="birthDate" label="Birth date" />
-              <TimeSpan name="age" label="Age" />
-              <Textarea name="aboutYourself" label="Tell us about yourself" height="100" />
-              <Button type="submit">Submit</Button>
-            </Stack>
+             {(formData) => (
+              <Stack gap="8">
```

Comment:
doesn't seem to be formatted. can you run `npm run prettier`?

Reply:
ran prettier.  has no effect